README.rst

```diff
@@ -0,0 +1,22 @@
+esi-ui
+======
+
+OpenStack Horizon plugin that supports ESI
+
+Installation
+------------
+
+Clone the repository::
+
+        git clone https://github.com/CCI-MOC/esi-ui.git
+
+Copy ``enabled`` file::
```

Comment:
Small nit; there are multiple files, and this command copies them all over

Reply:
The correct way of installing should be copying every file under the esi-ui/esi_ui/enabled directory into the horizon/openstack_dashboard/local/enabled directory, so I think this should be fine

Reply:
Yep - the nit is that this wording makes it sound like `enabled` is a file, and not a directory; you want to copy all the files in the `enabled` directory over.